core/global_config.cpp

```diff
@@ -236,13 +241,43 @@ bool GlobalConfig::_load_resource_pack(const String &p_pack) {
 	return true;
 }
 
+static String _find_project_file(DirAccess* p_da, bool p_res = false) {
+	p_da->list_dir_begin();
+	String ret = "";
+	while (true) {
+		bool isdir;
+		String f = p_da->get_next(&isdir);
```

Comment:
I know this is already done this way in various other places, but I think we should start to enforce using slightly more explicit variable names (i.e. no one-char names apart from iterations). Here `file` would be good IMO, maybe even `p_diraccess` too. WDYT?